governance/what-is-governance.md

```diff
@@ -0,0 +1,69 @@
+
+In true open source fashion, FIP0001 was adapted from Ethereum Improvement Proposal 1 [(EIP-1)](https://eips.ethereum.org/EIPS/eip-1), which was itself a legacy document based on prior open source governance processes. 
+
+Key elements of FIP0001 include the establishment of FIPs as the primary mechanism for change in the Filecoin ecosystem, as well as the use of community soft consensus for accepting proposals. 
+
+Soft consensus is a process whereby all participants must consent- explicitly, or implicitly- to the adoption of a proposed change.  Practically, this meant that FIPs that were ready for final consideration would enter a last call phase, which operated like a final notice period for the Filecoin community.  A FIP would be accepted so long as no one raised any substantive and/or previously unaddressed objections. 
+
+Soft consensus- unlike processes of hard consensus, such as voting- relies on an implicit bias towards accepting proposals.  When FIP0001 was first implemented, this bias reflected the anticipated function of the Filecoin governance process: proposals would suggest changes and updates to core protocol technology, and would thus require a high bar of expertise, many rounds of review, and explicit approval from Core Devs in order to be considered ready for last call. In other words, proposals would need to clear many technical hurdles in order to be considered final; and if they were, it was expected that there would be few reasons to oppose their acceptance. 
+
+This process was highly effective during Filecoin’s early development.  It enabled protocol engineers and implementation teams to build and enhance the Filecoin protocol in an open and documented way. However, as the Filecoin ecosystem began to grow, FIP0001 and the policy of soft consensus were challenged by new community proposals that proposed complex and unanticipated changes to the Filecoin protocol (see: FIP0036 and FIP0056).  Beyond proposing technical enhancements or improvements, these proposals sought to change network technical specifics in order to support changes to network cryptoeconomics or data storage business models. 
+
+When Filecoin and other early L1 projects were first being developed, there was a belief that protocols themselves should change rarely and slowly. As such, the soft consensus process was intentionally designed to introduce an incredibly high bar for FIP acceptance, with the early belief that the only changes that should be made to the protocol are those that are of categorical benefit to the entire network. 
+
+Over time, various protocol ecosystems began to challenge this assumption by adapting their governance systems to be more flexible, expansive, and adaptable. Following the rejection of FIP0036 and FIP0056, some stakeholders in the Filecoin ecosystem began to argue for a similar transition of Filecoin’s governance system. 
+
+### FIP00XX
+A draft of FIP00XX was first published in October 2023. 
```

Comment:
Rather than XX shall we say something like "a new FIP" in the documentation, and update this timeline?